app/views/charge.njk

```diff
@@ -123,7 +123,7 @@
                 <h2 class="govuk-heading-m">{{ __p("cardDetails.webPayments.payWithGooglePay") }}</h2>
 
                 {% set GooglePayButtonHTML %}
-                  <span class="govuk-visually-hidden">Google Pay</span>
+                  <img src="/images/google-pay-logo.svg" alt="Google Pay" />
```

Comment:
We don’t need the `/` — nobody pretends to do XML any more!

Reply:
Good spot - removed.